crm_request/models/request.py

```diff
@@ -49,8 +55,20 @@ def message_new(self, msg, custom_values=None):
         """
         if custom_values is None:
             custom_values = {}
-        custom_values.update({'description': msg.get('body')})
-        return super(CrmClaim, self).message_new(msg, custom_values)
+
+        defaults = {
+            'description': msg.get('body')
+        }
+
+        if 'partner_id' not in custom_values:
+            match_obj = self.env['res.partner.match.wp']
```

Comment:
Why do you use res.partner.wp instead of res.partner.match? wordpress module is not dependent of cmr_compassion.

Reply:
The remains from a copy-paste. Corrected